bitsail-connectors/bitsail-connectors-legacy/bitsail-connector-kafka/src/test/resources/kafka_to_print_pb_format.json

```diff
@@ -0,0 +1,52 @@
+{
+  "job": {
+    "common": {
+      "job_type": "STREAMING",
+      "job_plugin_lib_dir": "plugin",
+      "job_plugin_conf_dir": "plugin_conf",
+      "enable_dynamic_loader": true,
+      "instance_id": "1",
+      "internal_instance_id": "1",
+      "extra_properties": {
+        "update-mode": "append"
+      },
+      "proto": {
+        "descriptor": "",
```

Comment:
This json sample should be runnable. It's ok to add the pb info from the integration test you write.

Reply:
Added base64 encoded descriptor and class name